src/models/campaign.ts

```diff
@@ -13,6 +13,7 @@ export const metadataConfig = {
     categoryCodeForAntigens: "RVC_ANTIGENS",
     categoryComboCodeForTeams: "RVC_TEAMS",
     attibuteCodeForApp: "CREATED_BY_VACCINATION_APP",
+    attributeCodeForDashboard: "DASHBOARD_ID",
```

Comment:
Sometime in the future, we should rename those attributes to use always a `RVC_` for codes (RVC=Reactive Vaccination Campaign). I failed to do that for the first attribute. For now, let's keep them as they are.

Reply:
As discussed, let's do it now!